packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts

```diff
@@ -9,16 +9,19 @@ export type PasteComponentLeafOperationContext = {
   component: IComponentModel;
 };
 
-let copyTimes = 0;
-
 export class PasteComponentLeafOperation extends BaseLeafOperation<PasteComponentLeafOperationContext> {
   private componentCopy!: IComponentModel;
 
   do(prev: AppModel): AppModel {
+    let copyTimes = 0;
```

Comment:
If you move `copyTimes` to local scope, copyTimes will start from 0 every time user pastes. `copyTimes` should be global.

Reply:
Shared `copyTime` between components can cause copyTime to grow too fast and looks strangely     
For example，copy stack5 after copying text several times, the copytime becomes 13
<img width="292" alt="image" src="https://user-images.githubusercontent.com/96771399/227858987-e8cf0c64-5943-4de2-9384-9bf5447905f9.png">